### PR TITLE
introducing 3scale in the middleware integration list

### PR DESCRIPTION
**Pull request by maneta**
Adding 3scale to the middleware list.

I'm assuming here that the `tags` array will match with the OS template tags.

cc @spadgett

### CODE REVIEW

**Comment by spadgett** [comment]
> I'm assuming here that the tags array will match with the OS template tags.

That's right.

**Comment by spadgett** [comment]
Thanks, you'll need to build the dist files and amend your commit to add them. See

https://github.com/openshift/origin-web-catalog#contributing

**Comment by maneta** [comment]
@spadgett done the rebuild. thanks.

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "angular-schema-form-bootstrap": "^0.2.0",
     "angular-ui-bootstrap": "0.14.x",
     "bootstrap": "^3.3.6",
+    "bower": "^1.8.2",
```
Comment (spadgett) [inline package.json]: This should only be under `devDependencies`.

Comment (maneta) [reply package.json]: done in f56c3e74a924c61f4d2df236f5d00915cb971dac

**Pull request merged**